### DISCUSSION TITLE
Usage of EXTRA_FILES var

### DISCUSSION THREAD

**Discussion by ramanveerji** [nikhilbadyal/docker-py-revanced#507] [Q&A]
How to use EXTRA_FILES var in case the releases section has more than one file. For example, check this revanced gms core releases section, it has 2 releases. https://github.com/ReVanced/GmsCore/releases/tag/v0.3.1.4.240913

**Comment by nikhilbadyal** [comment]
Provide direct link to the apk in the EXTRA_FILES.

**Reply by ramanveerji** [reply]
What will happen on new build from gmscore then? Please test this.

**Reply by nikhilbadyal** [reply]
It will keep on sending same apks again & again. 

See the purpose of the builder is to build revanced apks and work not as GitHub/telegram uploader. So I'm not sure if it's worth thinking too much on EXTRA_FILES feature. I added it just for a minor convenience.

Also in your case,both the apks can be used as microg in your device. So i don't think it's worth the efforts. 

However I'm open to suggestions or any **issue** you are facing with current implementation.

**Reply by ramanveerji** [reply]
I'm using this
EXTRA_FILES=https://github.com/ReVanced/GmsCore/releases/latest@MicroG.apk

With this it pushes the latest release from gmscore. It is working great but the thing is now there are 2 different releases by gmscore so if I keep on using the same then it will push only 1 apk.

**Reply by nikhilbadyal** [reply]
Why do you need both the apks. When anyone of the 2 works.

**Reply by ramanveerji** [reply]
Because of those 2 releases auto builder goes mad. And thinks there is an update.

**Reply by nikhilbadyal** [reply]
That's an incorrect assumption. I use the same extra files setup as you and everything is fine with the builder. You can find my envs [here](https://github.com/nikhilbadyal/docker-py-revanced/blob/main/.env.my).

You probably have messed up something in your fork.

**Reply by ramanveerji** [reply]
Wait sometime for newer build.
